### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by josephwegner**
@hex337 README updates - let me know if anything is not super clear.

I would say this closes https://github.com/pushpop-project/pushpop/issues/16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+
+### Install
+
+1. [Download](https://github.com/pushpop-project/pushpop-starter/archive/master.zip) this starter project
+2. Unzip the package and place the pushpop-starter folder wherever you want it
+3. Open a terminal, and `cd` to your pushpop-starter repo
+4. `bundle install`
+5. To make sure everything is working, run `pushpop jobs:run_once --file=jobs/example_job.rb` - you should not see any errors.
+6. Start creating your jobs!
+
```
Comment (hex337) [inline README.md]: I would also have another section in here about how to have it run continuously in the background, and how to shut it down.

**Comment by josephwegner** [comment]
@hex337 Is 75ee00b kind of what you're thinking?

**Comment by hex337** [comment]
Yeah, that'll do.

**Comment by hex337** [comment]
:+1: :shipit:

**Pull request merged**